packages/vscode-extension/src/devices/DeviceBase.ts

```diff
@@ -20,6 +22,10 @@ export abstract class DeviceBase implements Disposable {
 
   abstract get lockFilePath(): string;
 
+  async reboot(): Promise<void> {
+    this.preview?.dispose();
```

Comment:
we should do `this.preview = undefined` here – disposed instances shouldn't be reused